discord/guild.py

```diff
@@ -1800,6 +1800,7 @@ async def create_forum(
         category: Optional[CategoryChannel] = None,
         slowmode_delay: int = MISSING,
         nsfw: bool = MISSING,
+        media_only: bool = MISSING,
```

Comment:
media only isn't really a term used anywhere for this, wouldn't just `media` be better?

Reply:
I guess? I personally think it's ambiguous; `media=True` could also be interpreted as meaning media is disallowed by default, and this toggle enables it.

But maybe `media_only` isn't best either, since you can still create posts without an image in these channels.

Reply:
`media` would follow the same pattern as `news` in text channels (since in both cases they're different channel types internally)

Reply:
Fair point. I'm going to change it.